enhancements/update/cvo-log-level-api.md

```diff
@@ -316,6 +327,29 @@ This alternative was not chosen due to it breaking the existing API consistency
 to an extent. `ClusterVersion` will continue to configure the cluster 
 version, and the new `ClusterVersionOperator` resource will configure the CVO.
 
+### The ClusterVersionOperator CRD will be applied to the management cluster
+
+As the hosted CVO is running in the management cluster, its configuration 
+resource will also be applied to the management cluster. The CRD will be 
+namespace scoped; one `ClusterVersionOperator` resource per one hosted 
+control plane namespace. The HyperShift will not have to process the 
+`ClusterVersionOperator` resource in the hosted cluster as there will be none. 
+This will result in less logic for the HyperShift to overwrite any user 
+changes, and the hosted CVO configuration will be maintained by the same API 
+server that maintains the hosted CVO.
+
+This alternative was not chosen, as it increases the overall complexity of the 
+solution. The HyperShift would have to support multiple potentially
+different versions of the CRD, as the HyperShift is able to handle multiple
+versions of OCP clusters on the same management cluster. The hosted CVO would 
+have to learn to process multiple API servers simultaneously and would be given 
+additional network and API server access in the management cluster.
+
+An alternative is to simply propagate the desired CVO configuration from 
+the `HostedCluster` API to the hosted `ClusterVersionOperator` resource. 
+This utilizes an existing pattern and makes the whole solution simpler and 
+thus less prone to errors and easier to maintain.
```

Comment:
What about the alternative of just having flags that the HyperShift operator configures for CVO, rather than having to rely on the CRD when in a hosted cluster?

Reply:
Yes, let me think about that. That would be perfectly acceptable for our log level case. However, I wonder for the future and any potential extensions of the `ClusterVersionOperator` CRD and parsing any potential complex structures into flags / environment variables / config files. Let me ponder about alternative approaches to getting the data into the hosted CVO without creating new CRDs in the management cluster.

Reply:
>  ..any potential extensions of the ClusterVersionOperator CRD and parsing any potential complex structures into flags ....

Although this may never come. And if it would come, we could solve it then.

Reply:
Adding an offline mode that reads the OCP ClusterVersionOperator CRD from file, rather than from the cluster, would be a good solution.

HCP could generate that, mount it in the CVO pod and point the CVO binary to this file on disk.

That would avoid having the structural issues you've suggested may come in the future, and is similar to the direction Kubelet and other controllers are moving

Reply:
I love the idea, thank you. A configuration file is newly being proposed. A ClusterVersionOperator manifest file is proposed as a potential solution for the configuration file.